webdev4.html

```diff
@@ -24,22 +24,126 @@
+	subdomains: 'abcd',
+	minZoom: 0,
+	maxZoom: 20,
+	ext: 'png'
+    }).addTo(map);
+        
+    // Add OpenStreetMap tile layer to map element, Changed Basemap
+    // L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', { attribution: '© OpenStreetMap' }).addTo(map);
+        
+    // load GeoJSON from an external file, $=JQuery, data=GeoJSON Data      
+    $.getJSON("1duluth_precincts_WGS84.geojson",function(data){
     
-    // Add OpenStreetMap tile layer to map element
-    L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', { attribution: '© OpenStreetMap' }).addTo(map);
+    // For searching geojson data
+    console.log('geojson data', data);
+        
+    // add GeoJSON layer to the map once the file is loaded
+    L.geoJson(data,{
+        style: function (feature) {
+
+            // Add more ** calculate party percent of vote using function
+            var PartyPct = calculatePartyPct(feature);
+            console.log(PartyPct);
+
+            // assign colors from the ColorBrewer yellow-green scale 
+            var fill;
+            
+            // equal interval classification
+            // rep 40%
+            if (PartyPct.rep >= 40) {
+                fill = '#ca0020';
+                console.log('rep40');
+                }
+            
+            // rep 30%
+            else if (PartyPct.rep >= 35) {
+                fill = '#f4a582';
+                console.log('rep35');
+                }
+            
+            // else party 
+            else if (PartyPct.rep <= 34 && PartyPct.dem <= 59) {
+                fill = '#f7f7f7';
+                console.log ('neutral vote');
+                }
+            
+            // dem 60
+            else if (PartyPct.dem <= 60) {
+                fill = '#92c5de';
+                console.log('dem60');
+                }
+        
+            // dem 65% or more 
+            else {
+                fill = '#0571b0';
+                console.log('dem65');
+                }
+
+            // created a divergingsymbology based the democrat or republic vote as a percent of the total vote.
+            // Used Colorbrewer's 5-class Red-Blue colors for this http://colorbrewer2.org/#type=diverging&scheme=RdBu&n=5
+
+            // return style specification: outline 
+            return {
+            color: '#636363',
+            weight: 2,
+            fillColor: fill,
+            fillOpacity: 0.7
+            }
+            }, // End of $.geoJSON function
+
+            // created a popup
+            onEachFeature: function (feature, layer) {
+            var PartyPct = calculatePartyPct(feature);
+            // show name of precinct and party percent in popup
+            layer.bindPopup(
+            "<h3>" + feature.properties.PCTNAME + "</h3>" +
+            "<p> Democratic Party Vote:" + PartyPct.dem + "%</p>" 
+                + "<p>Republican Party Vote:" + PartyPct.rep + "%</p>");
+            }
+            }).addTo(map);
+            });	
```

Comment:
I know you said this is how you prefer to see your code, but this sort of formatting makes it very difficult to tell which bracket ends where. Remember, indentation is your friend 😄